libdnf/rpm/reldep_list.cpp

```diff
@@ -93,8 +93,8 @@ bool ReldepList::add_reldep_with_glob(const std::string & reldep_str) {
 
     dataiterator_init(&di, pool, 0, 0, 0, dep_splitter.get_name_cstr(), SEARCH_STRING | SEARCH_GLOB);
     while (dataiterator_step(&di)) {
-        ReldepId id =
-            Reldep::get_reldep_id(pImpl->sack, di.kv.str, dep_splitter.get_evr_cstr(), dep_splitter.get_cmp_type());
+        ReldepId id = Reldep::get_reldep_id(
+            pImpl->sack.get(), di.kv.str, dep_splitter.get_evr_cstr(), dep_splitter.get_cmp_type());
```

Comment:
What about to get sack before the while loop? Then it will not depend on optimizations.

Reply:
Good point. I moved `sack.get()` before loop.